### DISCUSSION TITLE
Unexpected sorting behaviour in views

### DISCUSSION THREAD

**Discussion by ghost** [SkepticMystic/breadcrumbs#375] [Q&A]
I deactivated the alphabetical sorting in the views, but the resulting sorting order seems random. Example:
- "DailyNote" acting as parent, with transcluded links to A, B, C, D. Relationship is stated in the YAML of these four notes.
- A, B, C, D need to appear in this order in "DailyNote" irrespective of their real name (say, Pears, Apples, Prunes, Coconuts).
- A, B, C, D have been created with the Refactor plugin.
- A, B, C, D have a faux creation date, i.e. a date I changed in macOS Terminal after I finished bulk populating/refactoring notes in Obsidian from some other app. It was a necessary step to keep things sorted. That said, all notes appear correctly sorted by creation date both in the file explorer and in the backlinks pane.
- Both the matrix and the list views show instead A, C, D, B or some other jumbled order.
Is there a practical reason for this behaviour, i.e. their modified creation date, as I suspect? Or could it be a bug? Or am I interpreting the settings incorrectly? Thanks!

**Comment by michaelpporter** [comment]
Views sort by the **Sort** option you choose, not by file creation date — and there isn't a created/modified-time sort, so with sorting turned off you get the graph's internal edge order (effectively arbitrary), which is what you're seeing.

To force a fixed A→B→C→D order independent of filename, give each child a numeric frontmatter property and sort by it. For example set `order: 1`, `order: 2`, … on the children, then set the view's sort field to `neighbour-field:order`. That sorts neighbours by that property's value, giving you full manual control.

If you'd specifically like a creation-date sort option added, let me know and I'll track it as a feature request.
